### DISCUSSION TITLE
arrow with dot instead of connection?

### DISCUSSION THREAD

**Discussion by KMR1324** [freeplane/freeplane#738] [Help]
some connections in freeplane were replaced by a short arrow with a dot. How can I set freeplane to show the whole connection again?

**Comment by EdoFro** [comment]
That happens when one end of the connector is attached to a (momentary) not visible node (For example, it has been filtered or it's part of a folded branch)

in menu `View/View settings` are also settings that affects connectors:
![image](https://user-images.githubusercontent.com/35700575/195413297-3e3c80c6-fbaf-4886-87c8-7b90b40c9e8a.png)

**Comment by KMR1324** [comment]
Thank you very much 😊

Von: EdoFro ***@***.***>
Gesendet: Mittwoch, 12. Oktober 2022 19:53
An: freeplane/freeplane ***@***.***>
Cc: Katrin Müller-Riemenschneider ***@***.***>; Author ***@***.***>
Betreff: Re: [freeplane/freeplane] arrow with dot instead of connection? (Discussion #738)


That happens when one end of the connector is attached to a (momentary) not visible node (For example, it has been filtered or it's part of a folded branch)

in menu View/View settings are also settings that affects connectors:
[Das Bild wurde vom Absender entfernt. image]<https://user-images.githubusercontent.com/35700575/195413297-3e3c80c6-fbaf-4886-87c8-7b90b40c9e8a.png>

—
Reply to this email directly, view it on GitHub<https://github.com/freeplane/freeplane/discussions/738#discussioncomment-3863046>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A3PCPAB3WS4HHAZ3635OAM3WC33HTANCNFSM6AAAAAARDHK3XM>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/freeplane/freeplane/discussions/738#discussioncomment-3863046", "url": "https://github.com/freeplane/freeplane/discussions/738#discussioncomment-3863046", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
